app/data/mysql/account_store_test.go

```diff
@@ -11,10 +13,37 @@ import (
 func TestAccountStore(t *testing.T) {
 	db, err := mysql.TestDB()
 	require.NoError(t, err)
-	store := &mysql.AccountStore{db}
+	var store data.AccountStore = &mysql.AccountStore{db}
 	for _, tester := range testers.AccountStoreTesters {
 		db.MustExec("TRUNCATE accounts")
 		db.MustExec("TRUNCATE oauth_accounts")
 		tester(t, store)
 	}
+
+	t.Run("handle oauth email with null value", func(t *testing.T) {
+		db := store.(interface {
```

Comment:
Why do you need to redefine `db` here?  Seems like the one defined in l14 should work?

Reply:
It's because `data.AccountStore` does not implements `Exec` but `sqlx.Ext` does, we can replace that hardcoded interface for `sqlx.Ext` to make it clear. What do you think?

Reply:
@AlexCuse Good catch! Thanks for that